docs/source/usage/config/authentication.md

```diff
@@ -1,4 +1,9 @@
-# Authentication
+---
+title: "Authentication"
+sidebar_title: "Authentication"
+description: "Authenticating with Apollo Studio"
```

Comment:
Will our users know what Apollo Studio is? Perhaps "Link the CLI to Apollo Studio with an API Key" here is a better desc.